server/models/procedures/stock.sql

```diff
@@ -625,7 +625,7 @@ BEGIN
 
 
   SET _sum_days = DATEDIFF(_end_date,  _start_date) + 1;
-  SET _number_of_month = (DATEDIFF(_end_date,  _start_date) + 1)/30.5;
+  SET _number_of_month = ROUND(DATEDIFF(_end_date,  _start_date)/30.5);
```

Comment:
We need to figure out a better way to do this.  If we have 0.45 months, this will say 0 months.  Does that make sense?  I would think it should be at least 1.  Maybe we can use `CIELING()`?  Or, set _number_of_month to be `1` if it is `0`?